### PR TITLE
Cchs 301

### PR DESCRIPTION
**Pull request by cschroed-usgs**
Each tile retrieval is timed. The entire script can be timed by wrapping the script in a call to `time`:
```
time bash rasterCurls.sh https://cida-test.er.usgs.gov/dev/coastalchangehazardsportal _0b57c74f-d8d6-4686-a4fb-0de9b5c7f6d0 EWyGbdyH
```

### CODE REVIEW

File: seedAllRasters.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (cschroed-usgs) [inline seedAllRasters.sh]: entry point

**Comment by jiwalker-usgs** [comment]
I think this looks fine, my one comment would be to move them into one of the modules (portal or geoserver probably).  Having them at the root level seems slightly out of place.

**Comment by jiwalker-usgs** [comment]
Also, a comment for future improvement (if this ends up being a longer term solution) would be to only go by the item_id and add a step where it gets the wms url from the services section of that item.

**Comment by cschroed-usgs** [comment]
Thanks @jiwalker-usgs . I like your idea for improvement and I will move the scripts into a module.

File: coastal-hazards-portal/invalidateRasterCache.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (cschroed-usgs) [inline coastal-hazards-portal/invalidateRasterCache.sh]: Suzanne had the good idea of splitting invalidation out into a separate step. I will create a separate Jenkins job for seeding vs invalidating. I can create a parent job that calls both.

**Pull request merged**